docs/developer.rst

```diff
@@ -88,23 +85,20 @@ for more details. Install the following packages::
   pacman -S patch realpath
 
 
-Once everything is installed. Open PowerShell, make sure you've added MSYS2
-path to current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
-accessible. Activate conda environment. Folllowing command build with cuda
-enabled with PDB generated, adjust it to whatever suitable for you.::
+Once everything is installed. Open PowerShell, and make sure MSYS2 is in the
+path of the current session. Ensure ``bazel``, ``patch`` and ``realpath`` are
+accessible. Activate the conda environment. The folllowing command build with
+cuda enabled adjust it to whatever suitable for you::
 
   python .\build\build.py `
     --enable_cuda `
     --cuda_path='C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1' `
     --cudnn_path='C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1' `
-    --cuda_compute_capabilities='6.1' `
```

Comment:
I think it's fine to leave the CUDA flags in!

I just suggested removing the debug flags.

Reply:
I saw an outdated state. The flags look ok now.